packages/peregrine/lib/store/actions/checkout/asyncActions.js

```diff
@@ -317,10 +304,21 @@ export const createAccount = history => async (dispatch, getState) => {
 
 /* helpers */
 
-export function formatAddress(address = {}, countries = []) {
-    const country = countries.find(({ id }) => id === 'US');
+/**
+ * Formats an address in the shape the REST API expects.
+ * TODO: Can we remove this code once address submissions switch to GraphQL?
+ *
+ * This function may throw.
+ *
+ * @param {object} address - The input address.
+ * @param {object[]} countries - The list of countries data.
+ */
+export const formatAddress = (address = {}, countries = []) => {
     const { region_code } = address;
-    const { available_regions: regions } = country;
+
+    const usa = countries.find(({ id }) => id === 'US');
+    const { available_regions: regions } = usa;
```

Comment:
It's possible we could genericize this function not to be specific to "US" but Venia's UI remains specific to the USA so I didn't change this for now.

Reply:
More details: All we have from the form is a `region_code` (ex: "AL"). Since this could match a valid region in any number of countries, I don't think we should touch this code right now.